apps/web/src/pages/ApplicantDashboardPage/components/ReviewRecruitmentProcessPreviewList.tsx

```diff
@@ -160,6 +165,40 @@ const ReviewRecruitmentProcessPreviewList = ({
           </p>
         </Well>
       )}
+      <div
+        data-h2-margin-top="base(x1)"
+        data-h2-padding-top="base(x1)"
+        data-h2-border-top="base:all(1px solid gray.light)"
+      >
+        <p data-h2-font-weight="base(bold)" data-h2-margin-bottom="base(x.125)">
+          {intl.formatMessage({
+            defaultMessage: "Off-platform recruitment processes",
+            id: "tpXtAJ",
+            description: "Off-platform section header",
+          })}
+        </p>
+        <p data-h2-color="base(black.light)" data-h2-margin-bottom="base(x1)">
+          {intl.formatMessage({
+            defaultMessage:
+              "If you're qualified in processes or pools on other Government of Canada platforms, you can tell us here. This information will be verified.",
+            id: "AC/qwa",
+            description: "Off-platform section information",
+          })}
+        </p>
```

Comment:
A designer could clarify but, I beleive this is intended to also be the caption font size.

Reply:
I think you're right https://github.com/GCTC-NTGC/gc-digital-talent/pull/13007/commits/ae24fc25efacefe510c260c3e0a6edb7b596b461